### ISSUE TITLE
FeedIo\Feed\Item\Media::setLength(): Argument #1 ($length) must be of type ?string, int given, called in .../FeedIo/Parser/JsonParser.php on line 104

### ISSUE THREAD

**Issue by lesion** [alexdebril/feed-io#374]
while parsing a JsonFeed I receive the following error:

```
FeedIo\Feed\Item\Media::setLength(): Argument #1 ($length) must be of type ?string, int given, called in .../FeedIo/Parser/JsonParser.php on line 104
```

the spec says that has to be a number:
`size_in_bytes (optional, number) specifies how large the file is.`
https://www.jsonfeed.org/version/1.1/

and feed-io is using a string:
https://github.com/alexdebril/feed-io/blob/main/src/FeedIo/Feed/Item/Media.php#L15

thanks :heart:

**Comment by alexdebril** [comment]
Thanks for that feedback and thanks for the :heart: at the end of the message, I think we all need more of this. Actually, if everyone could send some love with more :heart: in every issue they raise, the open-source community would be the most lovable thing on the internet and we could get rid of all those cat pictures. Because we wouldn't need them anymore to "feel good", so we would kick all of them in the woods where there belong as we all know that in the real life, every single cat is a complete douche bag.

Anyway, of course I'll fix it very soon in a way that doesn't break backward compatibility.

**Comment by alexdebril** [comment]
@lesion could you share with me the URL of the JSON Feed please ?

**Comment by lesion** [comment]
https://radioblackout.org/feed/json

**Comment by alexdebril** [comment]
it's on the way for 4.9.x branch, I'm porting it to main branch right now

**Comment by alexdebril** [comment]
Fix published through versions 4.9.9 & 5.0.5
